release-notes/opensearch-ml-commons-dashboards.release-notes-2.7.0.0.md

```diff
@@ -0,0 +1,7 @@
+## Version 2.7.0.0 Release Notes
+
+Compatible with OpenSearch 2.7.0
+
+### Breaking Changes
+
+* ml-commons introduced a API breaking change in 2.7 which changed the API response data, FE is updated accordingly. With this change, FE will no longer compatible with OpenSearch version < 2.7 ([#154](https://github.com/opensearch-project/ml-commons-dashboards/pull/154))
```

Comment:
Could we adjust the description about this breaking change according [the conventional](https://github.com/opensearch-project/opensearch-plugins/blob/main/RELEASE_NOTES.md)?

Reply:
Could you elaborate a bit?

Reply:
I can't find the breaking changes example in other plugins. The description should short and use capitalized for first letter. I may change to 
```
* Change model_state according ml-commons 2.7 ([#154](https://github.com/opensearch-project/ml-commons-dashboards/pull/154))
```

Reply:
> The description should short

I think the description doesn't have to be "short", the description can be compact but at least it should shortly explain what's the change and why making this change, especially for breaking changes.

Also, I personally don't feel the breaking change example in documentation is a good example :D

> use capitalized for first letter

It should use capitalized letter when it applicable, but in this case, the description starts with `ml-commons` which is a component name